Build failure: Coloquinte

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

`nix-build -A coloquinte`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/314821261

### Relevant log output

```console
Running phase: buildPhase
build flags: -j24 SHELL=/nix/store/rlq03x4cwf8zn73hxaxnx0zn5q9kifls-bash-5.3p3/bin/bash
[  5%] Building CXX object CMakeFiles/coloquinte.dir/src/coloquinte.cpp.o
[  5%] Building CXX object CMakeFiles/coloquinte.dir/src/parameters.cpp.o
[  8%] Building CXX object CMakeFiles/coloquinte.dir/src/place_global/place_global.cpp.o
[ 11%] Building CXX object CMakeFiles/coloquinte.dir/src/place_detailed/legalizer.cpp.o
[ 14%] Building CXX object CMakeFiles/coloquinte.dir/src/place_global/net_model.cpp.o
[ 17%] Building CXX object CMakeFiles/coloquinte.dir/src/place_global/density_legalizer.cpp.o
[ 25%] Building CXX object CMakeFiles/coloquinte.dir/src/place_global/density_grid.cpp.o
[ 25%] Building CXX object CMakeFiles/coloquinte.dir/src/place_detailed/abacus_legalizer.cpp.o
[ 25%] Building CXX object CMakeFiles/coloquinte.dir/src/place_global/transportation_1d.cpp.o
[ 28%] Building CXX object CMakeFiles/coloquinte.dir/src/place_detailed/row_legalizer.cpp.o
[ 31%] Building CXX object CMakeFiles/coloquinte.dir/src/place_detailed/row_neighbourhood.cpp.o
[ 34%] Building CXX object CMakeFiles/coloquinte.dir/src/place_detailed/tetris_legalizer.cpp.o
[ 37%] Building CXX object CMakeFiles/coloquinte.dir/src/place_global/transportation.cpp.o
[ 40%] Building CXX object CMakeFiles/coloquinte.dir/src/place_detailed/place_detailed.cpp.o
[ 45%] Building CXX object CMakeFiles/coloquinte.dir/src/place_detailed/incr_net_model.cpp.o
[ 45%] Building CXX object CMakeFiles/coloquinte.dir/src/place_detailed/detailed_placement.cpp.o
/build/source/src/place_detailed/tetris_legalizer.cpp: In member function 'std::pair<bool, int> coloquinte::TetrisLegalizer::attemptPlacement(int, int) const':
/build/source/src/place_detailed/tetris_legalizer.cpp:48:20: error: 'clamp' is not a member of 'std'
   48 |     int pos = std::clamp(x, b, e);
      |                    ^~~~~
make[2]: *** [CMakeFiles/coloquinte.dir/build.make:191: CMakeFiles/coloquinte.dir/src/place_detailed/tetris_legalizer.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:133: CMakeFiles/coloquinte.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

### Additional context

This package hasn't seen any commits (except treewides) since its initialization in july 2023, it is outdated (most recent release is nov 2023) and the package name is `coloquinte` (the name of the upstream & nixpkgs maintainer), not `placeroute` (the repository name). The build failure is also listed over here: https://github.com/NixOS/nixpkgs/issues/459759

### System metadata

```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.58, NixOS, 25.11 (Xantusia), 25.11.20251117.89c2b23`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.93.3
System type: x86_64-linux
Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/jasper/.config/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/jasper/.nix-profile/etc/xdg/nix/nix.conf:/home/jasper/.local/state/nix/profile/etc/xdg/nix/nix.conf:/home/jasper/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/jasper/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/i6lm069c9din9v4ry6szspx5zrn0qzhv-lix-2.93.3/share`
 - nixpkgs: `/nix/store/f640ps0hcp7w5jzg18djf8gdhl6r2rnl-source`
```

### Notify maintainers

@Coloquinte 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/co/coloquinte/package.nix
+++ b/pkgs/by-name/co/coloquinte/package.nix
@@ -10,13 +10,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "coloquinte";
-  version = "0.3.1";
+  version = "0.4.3";
 
   src = fetchFromGitHub {
     owner = "coloquinte";
     repo = "PlaceRoute";
     rev = finalAttrs.version;
-    hash = "sha256-bPDXaNZCNBM0qiu+46cL/zH/41lwqHPqfqTzJaERgVQ=";
+    hash = "sha256-BQg2rVYe1wJOX7YnvgDVpmN6hwBJZKH0fxm+8HC8bvY=";
   };
 
   nativeBuildInputs = [

